pages/keplr/notification-page.js

```diff
@@ -1,9 +1,15 @@
 const approveButton = `button`;
 const copyAddress = 'Copy Address';
-const copyWalletAddressSelector = 'div.sc-dkzDqf div.sc-hKMtZM.sc-kDDrLX.cyoEAq.dkJSBQ'
+const copyWalletAddressSelector =
+  'div.sc-dkzDqf div.sc-hKMtZM.sc-kDDrLX.cyoEAq.dkJSBQ';
+const walletSelectors = {
+  AGORIC: 'img[alt="Agoric local"]',
+  COSMOS: 'img[alt="Cosmos Hub"]',
+};
```

Comment:
we should have a function like so
`const walletSelector = (chainName) => 'img[alt="${chainName}]'`
so that we can dynamically generate a selector, instead of manually adding a selector for every chain

Reply:
Great suggestion. Done.